extensions/resource-deployment/src/localizedConstants.ts

```diff
@@ -23,14 +22,11 @@ export const refresh = localize('azure.refresh', "Refresh");
 export const createNewResourceGroup = localize('azure.resourceGroup.createNewResourceGroup', "Create a new resource group");
 export const NewResourceGroupAriaLabel = localize('azure.resourceGroup.NewResourceGroupAriaLabel', "New resource group name");
 export const realm = localize('deployCluster.Realm', "Realm");
-export const unexpectedOptionsSourceType = (type: OptionsSourceType) => localize('optionsSourceType.Invalid', "Invalid options source type:{0}", type);
 export const unknownFieldTypeError = (type: FieldType) => localize('UnknownFieldTypeError', "Unknown field type: \"{0}\"", type);
+export const optionsSourceAlreadyDefined = (optionsSourceId: string) => localize('optionsSource.alreadyDefined', "options source with id:{0} is already defined", optionsSourceId);
```

Comment:
Again, if you're localizing strings please make sure they're formatted nicely and are something the user can find useful. `Options source with id : '{0}' is already defined` for example

(and below suggest using `Options Source` instead of OptionsSource`)

Reply:
Fixed.